uberfire-structure/uberfire-structure-backend/src/main/java/org/guvnor/structure/backend/organizationalunit/OrganizationalUnitServiceImpl.java

```diff
@@ -225,7 +228,19 @@ public OrganizationalUnit createOrganizationalUnit(final String name,
             newOrganizationalUnit = organizationalUnitFactory.newOrganizationalUnit(groupConfig);
             registeredOrganizationalUnits.put(newOrganizationalUnit.getName(),
                                               newOrganizationalUnit);
-
+           
+            ConfigProperties configProperties= new ConfigProperties(System.getProperties());
+            JGitFileSystemProviderConfiguration jGitFileSystemProviderConfiguration =new JGitFileSystemProviderConfiguration();
```

Comment:
This is a lack of abstraction. OrganizationUnit/space is only an abstract concept and shouldn't have any knowledge of jgit internals and also it's not their responsibility to create directories on .niogit.